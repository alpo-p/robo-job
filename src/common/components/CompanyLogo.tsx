import React from 'react'
import { Image } from 'react-native'

interface P {
  url: string
  enableStylingForDetailedView?: boolean
}

export default ({ url, enableStylingForDetailedView }: P) => (
  <Image
    style={{
      height: 112,
      width: 112,
      borderRadius: 56,
      borderWidth: enableStylingForDetailedView ? 4 : 1,
      borderColor: 'white',
      alignSelf: 'center',
      position: 'absolute',
      bottom: enableStylingForDetailedView ? -12 : 70,
    }}
    source={{ uri: url }}
  />
)
