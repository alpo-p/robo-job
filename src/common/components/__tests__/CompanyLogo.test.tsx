import React from 'react'
import renderer from 'react-test-renderer'
import CompanyLogo from '../CompanyLogo'

describe('CompanyLogo', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<CompanyLogo url="testUrl" />)
    expect(tree).toMatchSnapshot()
  })
})
