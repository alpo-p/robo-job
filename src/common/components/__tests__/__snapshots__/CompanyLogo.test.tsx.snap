// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompanyLogo should match snapshot 1`] = `
<Image
  source={
    Object {
      "uri": "testUrl",
    }
  }
  style={
    Object {
      "alignSelf": "center",
      "borderColor": "white",
      "borderRadius": 56,
      "borderWidth": 1,
      "bottom": 70,
      "height": 112,
      "position": "absolute",
      "width": 112,
    }
  }
/>
`;
